.github/workflows/build-new.yml

```diff
@@ -0,0 +1,75 @@
+name: Pybind11 CI
+
+on:
+  push:
+    branches:
+      - master
+      - wheels
```

Comment:
just curious about the plan for the wheels branch vs. master branch.

Reply:
I just wanted to trigger the CI before opening the PR. Unnecessary now.